test/services/katello/pulp/rpm_test.rb

```diff
@@ -57,7 +57,7 @@ def teardown
       end
     end
 
-    class RpmTest < RpmTestBase
+    class RpmTest < RpmTestBaseRpm
       def test_find
         package = Rpm.find(@package_id)
 
```

Comment:
Renaming this base class name to see if this will cause failure that should happen with the line Katello::Rpm.import_for_repository(@repo, true)